packages/vc-status-list-issuer-rest-api/package.json

```diff
@@ -13,8 +13,8 @@
   },
   "dependencies": {
     "@sphereon/ssi-express-support": "workspace:*",
-    "@sphereon/ssi-sdk-ext.did-utils": "0.24.1-unstable.82",
-    "@sphereon/ssi-sdk-ext.identifier-resolution": "0.24.1-unstable.82",
+    "@sphereon/ssi-sdk-ext.did-utils": "0.24.1-next.96",
+    "@sphereon/ssi-sdk-ext.identifier-resolution": "0.24.1-next.96",
```

Comment:
Please do no update individual packages. Every ssi-sdk-(ext) always has to be in sync in the whole project.

Reply:
reverted this (and a couple of other) version updates